CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide/experimental/AbstractTree.java

```diff
@@ -28,10 +28,10 @@
  *
  * @author Nova
  */
-abstract class AbstractTree {
+public abstract class AbstractTree {
 
-    static final int MAX_OBJECTS = 50;
-    static final int MAX_LEVELS = 4;
+    public static final int MAX_OBJECTS = 50;
+    public static final int MAX_LEVELS = 4;
```

Comment:
I think I'd make these protected (unless there's a reason to make them public)

Reply:
Done